pkg/controller/networkpolicy/networkpolicy_controller.go

```diff
@@ -1180,6 +1180,9 @@ func (n *NetworkPolicyController) getMemberSetForGroupType(groupType grouping.Gr
 	for _, ee := range externalEntities {
 		groupMemberSet.Insert(externalEntityToGroupMember(ee, true))
 	}
+	for _, node := range nodes {
+		groupMemberSet.Insert(nodeToGroupMember(node))
```

Comment:
Like a Pod member in an AppliedToGroup, we don't need NodeIP in Node member when it's in an AppliedToGroup, otherwise the code needs to keep a Node group up-to-date when a Node IP changes but actually it's never used.